nucleus/prediction.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
This import seems unused to me - can you elaborate on what it's needed for?

Reply:
This allows for the type annotations in this file to work properly.

Reply:
why would it be needed here but not other files (like `nucleus/annotation.py`)